os/net/ipv6/sicslowpan.c

```diff
@@ -910,11 +937,13 @@ compress_hdr_iphc(linkaddr_t *link_destaddr)
            NHC byte extra - before the next header here. This is due to
            next not being elided in that case. */
         if(!IS_COMPRESSABLE_PROTO(*next_hdr)) {
+          CHECK_BUFFER_SPACE(1);
```

Comment:
I guess this it the first critically needed check as we have gotten into the loop here?